src/components/Messages/Messages.jsx

```diff
@@ -0,0 +1,41 @@
+import { useDispatch, useSelector } from "react-redux";
+import { close, handle_obsolete } from "../../slices/MessagesSlice";
+import store from "../../store";
+import Snackbar from "@material-ui/core/Snackbar";
+import Alert from "@material-ui/lab/Alert";
+import AlertTitle from "@material-ui/lab/AlertTitle";
+import "./ConsoleInterceptor.js";
+
+// A component that displays error messages
+function Messages() {
+  const messages = useSelector(state => state.messages);
+  const dispatch = useDispatch();
+  // Returns a function that can closes a message
+  const handleClose = function (message) {
+    return function () {
+      dispatch(close(message));
+    };
+  };
+  return (
+    <div>
+      <div>
+        {messages.items.map((message, index) => {
+          return (
+            <Snackbar open={message.open} key={index}>
+              <Alert icon={false} severity={message.severity} onClose={handleClose(message)}>
+                <AlertTitle>{message.title}</AlertTitle>
+                {message.text}
+              </Alert>
+            </Snackbar>
+          );
+        })}
+      </div>
+    </div>
+  );
+  return res;
+}
+// Invoke repetedly obsolete messages deletion (should be in slice file but I cannot find a way to access the store from there)
+window.setInterval(() => {
+  store.dispatch(handle_obsolete());
+}, 100);
```

Comment:
@biwano question on this item. Do we need to fire a "messages/handle_obsolete" slice to redux every 100ms? Right now it just litters (IMO) the Redux inspector with activity (nothing an end user would notice, but as a dev it floods redux with a lot of extra activity to sift through). See screenshot below.

Can we fire this event once a minute? Or once every 10 minutes & still be able to remove stale messages on behalf of the user?

![screenshot](https://user-images.githubusercontent.com/80423742/133138709-22518ef4-d142-4535-8c4a-a827bc4b5a77.png)

Reply:
Ho yeah, we can increase this value to once every minute or more. It is just a security to make sure the DOM does not get flooded.